spec/integration/aws_server_certificate_spec.rb

```diff
@@ -4,97 +4,79 @@
-5bNmtwGE+Gbir34KA/Pe0Lg8t8Y8eHedgbaNPegzL/PW3yO+z31UDAWT8lOjN2pq
-3LfgUS/9nae8kGc6HwiJOvAdog9q+bQJuGWdxjZ7gw1+5+oOvQdq4APPcL3+vdGU
-9Y39tQylvKAovd9g80wXUHRb/r04hKfYJmo6cVlz4Acz0iamiRDGCpFj3MIdYmNX
-/iSbUUbu+KYzC0pNiNI4TXOdRGNSEcpHXAbmKK2HWR2w+U/duL3g4mzJDEbzI64g
-fKC80Q+tesBwrzWnkXYaTWj422bngXuISpfVpkcW4mUloxLcSTVQyVSsXTB/SCvH
-yHjuy615AoGBAPdWnnZiFZoYGtygkdUw3lFp5BGdxKY6nN9jAfJYE6rrJQSkwxCl
-yaYLPlS/Pcepl/SQxphmWFvipzNa9fe85Uko57Zy1EHX/eWLpnLOjOefKxna9UIg
-9xH/6o/fA0upQGDDRjUrHPnKodtulA/yuUqQrPLlf4qUCtoeaqsKoT5dAoGBANuM
-TXEFry4EcU9jN4HmkZzkdNLtnLgHrgVX5VZx1cRhbs3mXKfxGpKWUy72Vwww3XUT
-1sGelmYbsjMtB8NIZAyMeIdaqS7DsnAu1l4fHUs0RIHNa4pqszx5bi8iYo7PaN50
-dmk2VtyO6eszbjjCkhWReFrYzMXBnIh7DKpT6fyfAoGAR9mTwtQPbmoeM4U5l/LQ
-Qlo+dJeeLqPMOmBqilnnrLkOUeEDAW0HvQJ7IudDLSMpD1SXPGJOvLKE27hKx6LK
-AIyvcyK8Yjw6d1owCh3SdN6aCLLAmGs3GrV7EDw6mtoZ54ISfRN/IVkp17KxtEhQ
-Z0bL1uuwNzN2S5KWbgVyfckCgYBq556t8j6jAbbLGVzl/AfbhcL9EobFdbffEjWy
-KiwVO/xgdlOX02dFCb3nmDw6y3CKmeZw0XAauFHFaZ/mD1Hoal4mSpcnFlPFHIl1
-u2DgRcs2CXjcJaixQc/NU8j6ETTXXY5rHPKe41g0Fw8MxHGt3u/kDL1pbiVyr7gz
-GDlPsQKBgQC+eFyfQnmcJHtB+aLy++JZsHtw03vIgmjSXYJYtqeyOYFdDD8gxTPv
-M4qaRvofyGQ4CVSGScInEcc7S+ACWoRvNuc0APgS8UFmJ15/fpptdlA5irhrMLHA
-3aH/+CxfbU+w/3tOoKvF91HIhsF+QXAUm7XNssGqYAe7m1gQYfGy4Q==
------END RSA PRIVATE KEY-----
+MIIEpAIBAAKCAQEArF5emKC/+Lg5Y8GQgFXf4esqcSHTm4RqrhR8nDSnqikqziHJ
+z3ZK9wWB7hgFwtFvOHIxjl9RkAlcSVpyxvqUSzYMyVb8pka99Iy2za1U1sVVKwiy
+5J+EdWv2R6YTjZE+u2w7/RyE6TY/92U/8q9dEX0C0RB7bv8JacnLAKQ/q1Pptpkz
+pf4YaAA9Qe/E3gGYnPPq4ekxUf6sFtu32u4uhJ1fTNtpNEt6wELkdzy4rbDO8PHS
+lSP5opxaDma08FPrJF4foPl49XYl/XT0UYoyo83q047pnIAha4rHe/T0nPGZYbE0
+TJHrlI2Ij5+3T3ZQUQpdh+ArnwdmVaOkFw6wcQIDAQABAoIBAEz8TTXQPk3BQmiq
+sHaRZFBML4Wd/RwttVQQ9GL0JZqbjnHIp5FQnUTdId4Mvq33yrwkTLvxGMXDWIOu
+sSrsCkXZWzal8mv1lqveGVuduhG+yz5QQU5ZbNjhInt30q3dHG6rddOj5D0hLMq7
+XyduaZwBALwNp4O4xySHq3Ka6ZEESpnY5o0hjclS7hAsiFnSW1/jI+yxPg6iS5T2
+VzU92m1S/yixWn3jlqPRBjjt4YxCwoi+8D5c3wt/91N6hUWrIMFRHuvBfMCibWia
+zquD022jCTkeW8S385ErYVHMLmZvXDcL0lwekoMMpL9TlTkapggz0desfVKt2aB5
+mnpDPJECgYEA3uXxptejw1QPJhpuEdXnLEjsOTl3wvtHZIGZzsIwgoScFEjSbQgb
++06Z7wo6QXnYqJPwt8jbdlyYLogcaBboFIAb7XAOyndliUEffgL855WM3jY5j04G
+ON3zGNXNQnnsgvWevs+bv17ZbEyqV8UiiolDCF4xn4CnP1y1nDJe44UCgYEAxfdp
+aD+DvGcMIRa1mfaCVLrTCWyThAwKbLK9jJqmvj4JJIBwwMcBBsYarAfuzzyxRH9B
++QlNVkFdB4pME+mAWj7TQLXVK81RetqcL/7+yebMJ0MJPRGHXaQ+p2Nv8ZLoYMRj
+MDruVLTXOg8XuN1RW4c8SmauaE2KXaYNUuzZXv0CgYEAwTy38tyfrIcDWxUut2ep
+skrGABZCLVeK3Sc+IHFZfM1aQnufccbF+2h5KzLCrmDj48HdvnbzS/maNTzq45J4
+QM2PaJjtObmo3QUIOEZ+2oZcSYjY/dO2sTY5uh4ghLEOyboGlYWGkLG57JnKU60j
+9NZqtqZyfsUaOWQ2TeOdP8ECgYEAxW6J4T32096xag6L7pC6SmZIMg3m0LpxaxaX
+k7JouTKFS7IMwTW3AFpyHz+KG4QcBoQj94ofZvapIOv8E5+8MkSVyuONRbHuoOeE
+/RkCYbmbwUxJ2m2w4uL62VWCPxqURm2VvnQHXNM+Etkaf3O5v96PcmQVbFBovhzt
+DNbJssECgYBIAFB3J66cZH3zBfyanHwYMPlTupxUBTsN19UCAK7MZqA5kbqG7b7Q
+cKPrgR7gibLvaxj22lvD8uEICMGy6g8s3+PyDwcw5PNrKOdIkRrYQxTFtfs1JRCa
+6kM8q32f1I3qd31zbpoDBHq+Se2RSaLbmoIAFqDSCWyeUHydALo2kw==
+-----END RSA PRIVATE KEY-----      
```

Comment:
same as above as extra spaces..

Reply:
It's part of server certificate so can't remove it